src/Core/Context.idr

```diff
@@ -2474,6 +2474,9 @@ addImportedInc modNS inc
                   do recordWarning (GenericWarn ("No incremental compile data for " ++ show modNS))
                      defs <- get Ctxt
                      put Ctxt ({ allIncData $= drop cg } defs)
+                     -- Tell session that the codegen is no longer incremental
+                     when (show modNS /= "") $
```

Comment:
Does this check still want to be here now that idris rebuilds from ttc?

Reply:
It's still building from source, so it just considers the .ttc outdated if the .so is missing. I expect this to be triggered if a installed package is missing incremental data.  (It's possible it may consider those as having missing ttc, checking now. )

Reply:
Yeah, for the case where a library is non-incremental, it will print the message and drop the incremental build.  

I made a quick test project that uses `elab-util`. 

If `elab-util` is installed non-incrementally, I get this message during build and it outputs a working, non-incremental executable:
```
Warning: No incremental compile data for Language.Reflection.Pretty
```

If I then re-install an incremental build of `elab-util` and try again.  Idris now knows to rebuild the project's `.ttc` and gives me a working incremental build.

If I try with a non-incremental `elab-util` and the main branch compiler, I get: 
```
1/1: Building Main (src/Main.idr)
Warning: No incremental compile data for Language.Reflection.Pretty
Warning: Compiling hole Main.isEqual
dunham@peri elab % build/exec/elab
Exception: variable Main-main is not bound
```

Reply:
Ah Ok, I misunderstood, that makes sense.